pyproject.toml

```diff
@@ -21,7 +22,7 @@ google-search-results = "*"
 pytest = "*"
 llama-index = "~0.9.0"
 duckduckgo-search = "*"
-crewai = "^0.30.11"
+crewai = "^0.35.3"
```

Comment:
Looks unrelated to this PR?

Reply:
Crewai v 0.30.11 complains about langchain dependency, while 0.35.3 works.